cmd/snap/main.go

```diff
@@ -262,6 +263,9 @@ func Parser(cli *client.Client) *flags.Parser {
 			arg.Name = name
 			arg.Description = desc
 		}
+		if c.extra != nil {
```

Comment:
I think this and the struct warrants a comment.

Reply:
I'll add it in a followup :-)